src/openforms/js/components/admin/form_design/FormConfigurationFields.js

```diff
@@ -290,6 +292,44 @@ const FormConfigurationFields = ({
           onChange={event => onCheckboxChange(event, active)}
         />
       </FormRow>
+      <FormRow>
+        <Field
+          name="form.activateOn"
+          label={
+            <FormattedMessage
+              defaultMessage="Activate on"
+              description="Form activation field label"
+            />
+          }
+          helpText={
+            <FormattedMessage
+              defaultMessage="When the form should be activated."
+              description="Form activation field help text"
+            />
+          }
+        >
+          <DateTimeInput value={activateOn} onChange={onChange} enableTime={true} />
```

Comment:
Not sure how easy to do, but I think that the widget should grey out datetimes in the past?